ui/redux/rootReducer.js

```diff
@@ -86,6 +86,19 @@ export const loadUserOptions = () => {
   }
 }
 
+export const loadStaffOptions = () => {
```

Comment:
So this function is identical to `loadUserOptions` except that the url is a tiny bit different. I would update this so `loadUserOptions` accepts a `staffOnly` parameter and conditionally creates the url. The you can define `loadStaffOptions` as 
```
export const loadStaffOptions = () => loadUserOptions(true)
```
The advantage to this is if for some reason in the future I need to change some of the behavior for fetching pr receiving users I only need to change it in one place instead of 2